java/arcs/core/storage/StorageProxy.kt

```diff
@@ -257,7 +257,26 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
         }
     }
 
-    private suspend fun notifyUpdate(data: T) = applyCallbacks(::onUpdateActions) { it(data) }
+    /** Safely make a copy of the specified action set, and launch each action on a coroutine */
```

Comment:
To channel Cameron: "full stop"
```suggestion
    /** Safely make a copy of the specified action set, and launch each action on a coroutine. */
```